xsd/netex_part_2/part2_journeyTimes/netex_coupledJourney_version.xsd

```diff
@@ -210,7 +210,7 @@ of the corresponding VEHICLE TYPE. true for forward.</xsd:documentation>
 			<xsd:documentation>Reference Elements for JOURNEY PART.</xsd:documentation>
 		</xsd:annotation>
 		<xsd:sequence>
-			<xsd:element name="MainPartRef" type="JourneyPartRefStructure">
+			<xsd:element name="MainPartRef" type="JourneyPartCoupleRefStructure" minOccurs="0">
```

Comment:
```suggestion
			<xsd:element name="MainPartRef" type="JourneyPartRefStructure" minOccurs="0">
```

Reply:
I agree with this suggestion

Reply:
and accept the PR provided that the suggestion is validated

Reply:
Do you want me to commit it, and continue? I am actually inclined to apply it to next.